gprofiler/main.py

```diff
@@ -528,6 +534,15 @@ def parse_cmd_args():
         "--continuous", "-c", action="store_true", dest="continuous", help="Run in continuous mode"
     )
 
+    parser.add_argument(
+        "--legacy-profile-api-version",
```

Comment:
This is one long argument name... Can you make it shorter?

Reply:
Solving

Reply:
Still long if you ask me. lol. But OK :)